src/components/NearMe/NearMeProfiles.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react'
+import { Box, ImageList, ImageListItem, Typography } from '@mui/material'
+import { LocationOn } from '@mui/icons-material'
+import { makeStyles } from 'tss-react/mui'
+
+type isMobileProps = {
+  isMobile: boolean
+}
+type objectType = {
+  id: string
+  location: string
+  picture: string
+  fName: string
+  lName: string
+}
+const NearMeProfiles = ({ isMobile }: isMobileProps) => {
+  const { classes } = useStyles()
+  const [list, setList] = useState<Array<objectType>>([])
+  const columns = isMobile ? 3 : 6
+  const mrgLeft = isMobile ? 1 : 5
+  const txtAlign = isMobile ? 'center' : 'left'
+  const getData = () => {
```

Comment:
all API calls go to folder `actions`, it's principle to split up API, logic and visual part.
We don"t use `then` callbacks instead `axios`

Reply:
The data export function from the file, which will later fetch data from the server, has been moved to a separate file userServices.ts in the actions directory.